Controller/Payment/Order.php

```diff
@@ -36,13 +36,23 @@ public function __construct(
 
         $this->checkoutFactory = $checkoutFactory;
         $this->catalogSession = $catalogSession;
+	$this->config = $config;
     }
 
     public function execute()
     {
         $amount = (int) (round($this->getQuote()->getBaseGrandTotal(), 2) * 100);
 
         $receipt_id = $this->getQuote()->getId();
+	    
+	$payment_action = $this->config->getPaymentAction();
+	    
+	if($payment_action == "authorize") {
```

Comment:
please add proper indentation and coding standards

Reply:
Fixed the indentation and coding standard problem.